contrail/functions

```diff
@@ -1,6 +1,19 @@
 
 with_venv=tools/with_venv.sh
 
+declare -A sku_dict
+sku_dict['2014.1']='icehouse'
+sku_dict['2014.2']='juno'
+sku_dict['2015.1.2']='kilo'
+sku_dict['12.0']='liberty'
+sku_dict['13.0']='mitaka'
+
+function get_sku() {
+    ver=`${with_venv} nova-manage version`
```

Comment:
This adds a dependency that the tempest script has to be run from the openstack node.
Do we have a better option? Shall we use the client version, moreover its the cli option?

Also the mapping isnt right. The version output should be trimmed to single dotted version.
root@b3s10:~# ver=`nova-manage version`
root@b3s10:~# echo $ver
2014.2.3
root@b3s10:~# echo ${sku_dict[${ver}]}

root@b3s10:~#